src/content/docs/integrations/browser.mdx

```diff
@@ -59,7 +59,7 @@ import { App } from './App'
 
 async function enableMocking() {
   if (process.env.NODE_ENV !== 'development') {
-    return
+    return Promise.resolve()
```

Comment:
You don't have to return an explicit `Promise.resolve()` here. 

Since the `enableMocking` function is marked as `async`, it will always return a Promise. You can chain to that promise even if this condition is reached. 

Returning `Promise.resolve()` has a [different effect](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve).